spec/deaf_grandma_spec.rb

```diff
@@ -9,10 +9,18 @@
   end
   
   it "says 'NOT SINCE 1964!' when we yell" do
-    #implement your test here
+    expect(script.speak("HI GRANDMA")).to eq "NOT SINCE 1964!"
   end
 
   it "EXTRA CREDIT: How would you test yelling BYE?" do
-    #implement your test here
+    if @bye_counter == 3
```

Comment:
Not sure this is the way you'd want to run this test, but regardless, keep in mind that you need to test for more than one thing with yelling BYE to grandma.